channels/pk.m3u

```diff
@@ -19,8 +19,6 @@ http://imob.dunyanews.tv/live/_definst_/dunyalive_1/chunklist_w1239250459.m3u8
 http://imob.dunyanews.tv/live/_definst_/dunyalive_1/FluxusTV.m3u8
 #EXTINF:-1 tvg-id="DunyaNews.pk" tvg-name="Dunya News" tvg-country="PK" tvg-language="Urdu" tvg-logo="" group-title="News",Dunya News (360p) [Not 24/7]
 https://imob.dunyanews.tv/live/_definst_/dunyalive_1/playlist.m3u8
-#EXTINF:-1 tvg-id="DunyaNews.pk" tvg-name="Dunya News" tvg-country="PK" tvg-language="Urdu" tvg-logo="" group-title="News",Dunya News (360p) [Offline]
-https://imob.dunyanews.tv/live/_definst_/ngrp:dunyalive_1_all/playlist.m3u8
```

Comment:
Not 24/7

![image](https://user-images.githubusercontent.com/30985701/132142190-b31475b7-c9c9-4243-ab84-da42f5be5cd2.png)

Adaptive being https://imob.dunyanews.tv/live/ngrp:dunyalive_1_all/playlist.m3u8